build.sh

```diff
@@ -413,7 +349,7 @@ isMSBuildOnNETCoreSupported()
             UNSUPPORTED_RIDS=("debian.9-x64" "ubuntu.17.04-x64")
```

Comment:
I believe this list is out of date.

Reply:
This is out of date; however, was never actually being checked as __DistroRid would used to be set after this was called. That being said, I removed debian-9, we can re-visit this list in the future.

Reply:
> however, was never actually being checked as __DistroRid would used to be set after this was called

This is strange. I've bumped into this about a week ago when I was trying to build coreclr on Debian 9 and it didn't build the managed stuff unless I've passed in the msbuildonunsupportedplatform option.

Reply:
Interesting, there must be another place where __DistroRid was being set then :\

Reply:
Will investigate.

Reply:
Addressed